notebooks/work-in-progress/fuel-price-imputation/fuel-price-imputation-zane.ipynb

```diff
@@ -0,0 +1,1080 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    frc_train_test = frc[frc.fuel_cost_per_mmbtu.notna()]
</pre><p>Mentioned this in Slack, but you probably want to use something like sklearn <code>train_test_split()</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/1696/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
I might not understand what `train_test_split()` does. I thought it split up a bunch of records that you know the target value for into different chunks for training vs. testing on, which is also something that's also built into the GridSearchCV?

Here I was trying to separate the records we **do** know the target value for from those where the target value is missing (since in this imputation problem they all show up in the same table to begin with). So I thought `frc_train_test` was the set of all records to use in training and testing, and the remaining records are the ones where we don't know the correct answer and we want to eventually generate one using the trained model.